config/docker/rustc-1.62.jinja2

```diff
@@ -24,11 +24,10 @@ RUN tar -xf ${RUST_TRIPLE}.tar.gz  -C /tmp/ && \
     rm -rf /tmp/${RUST_TRIPLE} && \
     rm /${RUST_TRIPLE}*
 
-# This image is based on clang-*jinja2 which only has clang installed but rustc
-# defaults to linker "cc" which calls the non-existing GNU stack (gcc, libgcc).
-# So we point cargo/rustc to use the LLVM stack via the clang linker entrypoint.
-RUN mkdir -p ${CARGO_HOME} && \
-    printf '[build]\nrustflags = ["-C", "linker=clang"]' > ${CARGO_HOME}/config.toml
+# This image is based on clang-*jinja2 which only has clang installed, but rustc
+# defaults to linker "cc" which usually points to the GNU/GCC stack. Creating
+# this symlink ensures both cargo and kernel build rustc invocations use clang.
+RUN ln -s $(which clang) /usr/bin/cc
```

Comment:
Sounds like the Debian `update-alternatives` tool should be used for that instead.  I think this is OK, I'm just a bit weary of pulling the rug under the distro's standard tooling.  I mean, normally you get this:

```sh
$ ls -l /usr/bin/cc
lrwxrwxrwx 1 root root 20 May 30  2022 /usr/bin/cc -> /etc/alternatives/cc
$ ls -l /etc/alternatives/cc
lrwxrwxrwx 1 root root 12 May 30  2022 /etc/alternatives/cc -> /usr/bin/gcc
```
which could (should?) be changed to the path to `clang` with `update-alternative install [...exercise for the reader...]`.